lib/launcher.ts

```diff
@@ -3,7 +3,6 @@
  * input configuration and launching test runners.
  */
 import * as fs from 'fs';
-import * as q from 'q';
 
 import {Config} from './config';
 import {ConfigParser} from './configParser';
```

Comment:
on line 14: `import * as helper from './util';`
But we use here just `runFilenameOrFn_` function, so we can import just it `import { runFilenameOrFn_ } from './configParser';`

Reply:
Done. Also updated runner with the same fix.